tests/use_cases/test_next_track_request_objects.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
`pytest` and imported classes from `request_object` not used, please erase. 

Also, check PEP8.